data/cncf.yaml

```diff
@@ -2673,10 +2716,11 @@
   maturity: sandbox
   repositories:
     - name: armada
-      url: https://github.com/G-Research/armada
+      url: https://github.com/armadaproject/armada
       check_sets:
         - community
         - code
+        - docs
```

Comment:
The `docs` check set is a subset of the `code` check set, so it isn't needed in this case. Please remove it.